react-frontend/src/lib/constants.ts

```diff
@@ -0,0 +1,93 @@
+interface Constant {
+    name?: string, 
+    replacement?: string
+}
+
+const constants: {[abbrev: string]: Constant } = {
```

Comment:
One more comment... I'm assuming the this list is based on documentation somewhere. It might be nice to include a link to that documentation so folks can see if new entries have been added in the future.